galaxy_ng/app/management/commands/set-repo-keyring.py

```diff
@@ -83,14 +90,14 @@ def handle(self, *args, **options):
             self.echo(f"Task failed with error: {task.error}", self.style.ERROR)
             sys.exit(1)
 
-        if AnsibleRepository.objects.get(pk=repo.pk).keyring == keyring:
+        if AnsibleRepository.objects.get(pk=repo.pk).gpgkey == pubkey:
```

Comment:
The field, now labeled "gpgkey," has an entry of the plain text public key, the output of filepath submitted to the subprocess command.